specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/BackupLocations.json

```diff
@@ -226,17 +226,23 @@
                     "description": "Password to access the location.",
                     "type": "string"
                 },
-                "encryptionKeyBase64": {
-                    "description": "Encryption key.",
+                "encryptionCertBase64": {
```

Comment:
I am not 100% sure on how certs work, but I know there is usually a public and private key.  Does this relate to the public key?  Private key?